CHANGELOG.md

```diff
@@ -14,15 +14,12 @@
 - PR #4088 Added asString() on ColumnVector in Java that takes a format string
 - PR #4040 Add support for n-way merge of sorted tables
 - PR #4053 Multi-column quantiles.
-- PR #4100 Add set_keys function for dictionary columns
```

Comment:
Looks like a potentially bad changelog merge?

Reply:
changelog updated.

Reply:
Yeah, that should be fixed though. I'll confirm, I did have a messy merge and an IDE file caching issue with it. Looks good now.